Headers/species_database_mod.F90

```diff
@@ -838,6 +838,21 @@ SUBROUTINE Init_Species_Database( Input_Opt, SpcData, SpcCount, RC )
 32  FORMAT( a30, " | ", 3f10.2 )
 40  FORMAT( a30, " | ", i10    )
 
+    !=======================================================================
+    ! Print metadata for only the species that are defined in this
+    ! simulation (but not the entire species database) to another YAML
+    ! file.  This facilitates coupling GEOS-Chem to external models.
```

Comment:
I don't think we need to put that this facilitates coupling to external models since I think the new file will be useful as a reference for any GEOS-Chem user.

Reply:
That's fine.  I'll make the edit.

Reply:
@lizziel The comment has been edited in 05704322001daf770881422339666ea3594b6caf.